coss/wsgi.py

```diff
@@ -13,6 +13,9 @@
 
 from django.core.wsgi import get_wsgi_application
 
+from raven.contrib.django.raven_compat.middleware.wsgi import Sentry
+
+
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "coss.settings")
 
-application = get_wsgi_application()
+application = Sentry(get_wsgi_application())
```

Comment:
Why do we need this? Is this a wagtail specific thing? In general just including the installed apps is enough for raven to send data to sentry.

Reply:
It is, this is just something extra:
https://docs.sentry.io/clients/python/integrations/django/#wsgi-middleware